ios photo collection 1.16/ios photo collection 1.16/View Controllers/PhotosCollectionViewCell.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+
+class PhotosCollectionViewCell: UICollectionViewCell {
+    
+    var photo: Photo? {
+        didSet {
+            updateView()
+        }
+        }
+    
+    func updateView() {
+        
```

Comment:
After passing `photo` to here, you need to unwrap it and assign it's values to your your outlets
```suggestion
        guard let photo = photo else { return }
        let newImage = UIImage(data: photo.imageData)
        imageView.image = newImage
        labelOutlet.text = photo.title
```